WordPress/Classes/ViewRelated/Site Creation/Wizard/SiteCreationWizardLauncher.swift

```diff
@@ -7,7 +7,7 @@ final class SiteCreationWizardLauncher {
     }()
 
     let steps: [SiteCreationStep] = {
-        if RemoteFeatureFlag.plansInSiteCreation.enabled() {
+        if RemoteFeatureFlag.plansInSiteCreation.enabled() && !AppConfiguration.isWordPress {
```

Comment:
[Nit] I think this is more readable:
```suggestion
        if RemoteFeatureFlag.plansInSiteCreation.enabled() && AppConfiguration.isJetpack {
```